raw_templates.go

```diff
@@ -1,3 +1,4 @@
 package main
-
+// This file is automatically generated by generate.go
+// Do not edit manually
```

Comment:
Shouldn't this be at the top of the file? Isn't that how `godoc` works?

Reply:
I thought `go generate` generated these for you, also: https://blog.golang.org/generate